packages/gitea/zarf.yaml

```diff
@@ -67,6 +67,17 @@ components:
           - gitea-values.yaml
     actions:
       onDeploy:
+        before:
+          - description: Check that the cluster has a storage class
+            shell:
+              windows: pwsh
```

Comment:
Pretty sure this won't work in `pwsh`
![image](https://github.com/defenseunicorns/zarf/assets/3977569/8ace1608-3ad3-441b-aed6-8d219e2817ef)

Reply:
For this stuff we may want to rethink windows support and how we could implement it.  A few ways off the top that might be useful:

1. Mutate if statements in some Powershell friendly way
2. Implement a conditional action kind that can handle this more in Zarf-land

Reply:
What do you think about doing this in the Zarf codebase instead of using shell commands in an action? My thinking is that we wouldn't have to do any tricks to make it work on Windows with that approach